modules/cnv_viewing_guest_ip_addresses.adoc

```diff
@@ -0,0 +1,59 @@
+[[viewing-guest-ip-addresses]]
+=== Viewing guest IP addresses
+
+You can view the IP addresses assigned to your Linux virtual machines by
```

Comment:
s/assigned/that are assigned

Reply:
Sounds good to me.

Reply:
It's an IBM Style point that one of my editors beat into me:
> Do not omit the word _that_ from clauses. The use of the conjunction _that_, although technically optional in some sentences, is never wrong and makes the sentence easier to
translate and clearer for readers whose primary language is not English. For example,
write “Verify that your directory service is working” instead of “Verify your directory
service is working.”

Reply:
Thanks for pasting that excerpt! I'll try to train my brain to remember "that." ;)